documentation-site/examples/select/label.js

```diff
@@ -6,13 +6,13 @@ import type {ValueT} from 'baseui/select';
 
 const ColorSwatch = styled<{$color: string}>('div', props => {
   return {
+    ...props.$theme.borders.border400,
```

Comment:
wonder if we should start turning everything into exact objects instead

Reply:
👍 updated